omnibus/config/software/openssl3.rb

```diff
@@ -33,17 +33,10 @@
 
 build do
   if !fips_mode?
-    installation_dir = if windows? then python_3_embedded else "#{install_dir}/embedded" end
-    command_on_repo_root "bazelisk run -- @openssl//:install --destdir=#{installation_dir}"
-    if windows?
-      block do
-        # shutil generates temporary files during run install that are not removed afterwards.
-        Dir.glob("#{installation_dir}/include/openssl/tmp*").each do |tmp_file|
-          delete tmp_file
-        end
-      end
-    else
-      lib_extension = if linux_target? then ".so.#{version}" else ".#{version}.dylib" end
+    # OpenSSL on Windows now gets installed as part of the Python install, so we don't need to do anything here
```

Comment:
does that mean that we effectively need to depend on `python3` for software that needs openssl on windows (well, that's only python so far, but just to confirm)

Reply:
it looks related but I'm not sure it really answers it

Reply:
That slack message was indeed addressing something else.

Yes, the idea is that on Windows, we only need OpenSSL for Python, and Python expects to find it in a specific location (essentially, Python "owns" the only copy of OpenSSL that we ship). If we were to depend upon it in any other way, we'd need to find a good way to have all dependants locate OpenSSL and we'd revisit this.  Unless I'm wrong here in my assumptions, of course.

This is, of course, ignoring the extra OpenSSL copies that some python deps may bring, which wouldn't be relevant for this point.